### ISSUE TITLE
Tests for WCAG technique H75: Ensuring that Web pages are well-formed

### ISSUE THREAD

**Issue by jessebeach** [quailjs/quail#87]
This issue will most likely result in several tests. Please open more issues and link them as appropriate.

Most likely we will want to use a third-party library for this issue. This library will need to be MIT license compatible.
## Testing

From http://www.w3.org/TR/WCAG20-TECHS/H75.html
### Procedure
1. Load each file into a validating XML parser.
2. Check that there are no well-formedness errors.
### Expected Results

Step 2 is true.

If this is a sufficient technique for a success criterion, failing this test procedure does not necessarily mean that the success criterion has not been satisfied in some other way, only that this technique has not been successfully implemented and can not be used to claim conformance.
